### DISCUSSION TITLE
what if i accidentally deleted my authentication app?

### DISCUSSION THREAD

**Discussion by calidion** [community/community#68142] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Can github give an option to disable it for those don't need such a feature?

**Comment by tiagoangelos** [comment]
I don't know about this option to deactivate, but it would be better if you requested a new authentication check so the old one will no longer be an option, I hope I helped:

visit: https://github.com/settings/two_factor_authentication/setup/intro

**Reply by calidion** [reply]
I mean the two factor authentication should not be mandated for users or projects are of no importance.
